xapian-bindings/go/docs/examples/index/index1.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import ("fmt"
+	"io"
+	"encoding/csv"
+	"os"
+	"xapian"
+)
+
+func main(){
+	//a csv reader to get the values from csv file
+	csvfile,err := os.Open("../data/100-objects-v2.csv")
```

Comment:
We don't want to add this data here as it's unfortunately under a Creative Commons "Attribution-NonCommercial-ShareAlike" licence which doesn't allow commercial use.  For example, I'd have to repack the source archive to remove it for the Debian packages (I maintain the Debian packages of Xapian).

I'd suggest it would make more sense to just translate the "simple" examples here (like is done for other languages) and add the more sophisticated examples which use this data to the "Getting Started" guide.